### ISSUE TITLE
nzbplanet

### ISSUE THREAD

**Issue by jbravo2uk** [theotherp/nzbhydra2#362]
since the update on nzbhydra nzb planet is not showing any movie cover art on searches.

**Comment by theotherp** [comment]
Please post you debug infos...

**Comment by jbravo2uk** [comment]
2019-04-16 04:42:26.465  INFO --- [0.0-5076-exec-2] org.nzbhydra.api.ExternalApi             : [Search: 571810, IP: <IP>] Received external API call: NewznabParameters{t=MOVIE, cat=[2000], offset=0, limit=100, raw=false, o=XML, attrs=[], extended=true}
2019-04-16 04:42:26.473  INFO --- [0.0-5076-exec-2] org.nzbhydra.api.ExternalApi             : [Search: 530516, IP: <IP>] Executing new search
2019-04-16 04:42:26.484  INFO --- [0.0-5076-exec-2] o.n.searching.IndexerForSearchSelector   : [Search: 530516, IP: <IP>] Selected 4 out of 4 indexers: Nzb.su, nzbgeek, nzbplanet.net, dog
2019-04-16 04:42:26.485  INFO --- [ool-61-thread-3] org.nzbhydra.indexers.Newznab            : [Search: 530516, IP: <IP>] dog: Calling https://api.dognzb.cr/api?apikey=<apikey>&t=search&extended=1&cat=2000&limit=100&offset=0
2019-04-16 04:42:26.485  INFO --- [ool-61-thread-2] org.nzbhydra.indexers.Newznab            : [Search: 530516, IP: <IP>] nzbplanet.net: Calling https://api.nzbplanet.net/api?apikey=<apikey>&t=search&extended=1&cat=2000&limit=100&offset=0
2019-04-16 04:42:26.485  INFO --- [ool-61-thread-4] org.nzbhydra.indexers.Newznab            : [Search: 530516, IP: <IP>] nzbgeek: Calling https://api.nzbgeek.info/api?apikey=<apikey>&t=search&extended=1&cat=2000&limit=100&offset=0
2019-04-16 04:42:26.485  INFO --- [ool-61-thread-1] org.nzbhydra.indexers.Newznab            : [Search: 530516, IP: <IP>] Nzb.su: Calling https://www.nzb.su/api?apikey=<apikey>&t=movie&extended=1&cat=2000&limit=100&offset=0
2019-04-16 04:42:26.990  INFO --- [ool-61-thread-2] org.nzbhydra.indexers.Newznab            : [Search: 530516, IP: <IP>] nzbplanet.net: Successfully executed search call in 493ms with 100 results
2019-04-16 04:42:28.098  INFO --- [ool-61-thread-4] org.nzbhydra.indexers.Newznab            : [Search: 530516, IP: <IP>] nzbgeek: Successfully executed search call in 1603ms with 100 results
2019-04-16 04:42:28.542  INFO --- [ool-61-thread-1] org.nzbhydra.indexers.Newznab            : [Search: 530516, IP: <IP>] Nzb.su: Successfully executed search call in 2037ms with 100 results
2019-04-16 04:42:28.626  INFO --- [ool-61-thread-3] org.nzbhydra.indexers.Newznab            : [Search: 530516, IP: <IP>] dog: Successfully executed search call in 2135ms with 100 results
2019-04-16 04:42:28.657  INFO --- [0.0-5076-exec-2] org.nzbhydra.searching.Searcher          : [Search: 530516, IP: <IP>] Returning results 1-100 from 380 results in cache. A total of 614421 results is available from indexers of which 0 were already rejected and 20 were removed as duplicates
2019-04-16 04:42:28.665  INFO --- [0.0-5076-exec-2] org.nzbhydra.api.ExternalApi             : [Search: 530516, IP: <IP>] Search took 2192ms. Returning 100 results
2019-04-16 04:42:28.875  INFO --- [0.0-5076-exec-4] org.nzbhydra.api.ExternalApi             : [Search: 335681, IP: <IP>] Received external API call: NewznabParameters{t=MOVIE, cat=[2000], offset=0, limit=100, raw=false, o=XML, attrs=[], extended=true}
2019-04-16 04:42:28.875  INFO --- [0.0-5076-exec-4] org.nzbhydra.api.ExternalApi             : [Search: 862915, IP: <IP>] Executing new search
2019-04-16 04:42:28.878  INFO --- [0.0-5076-exec-4] o.n.searching.IndexerForSearchSelector   : [Search: 862915, IP: <IP>] Selected 4 out of 4 indexers: Nzb.su, nzbgeek, nzbplanet.net, dog
2019-04-16 04:42:28.879  INFO --- [ool-67-thread-1] org.nzbhydra.indexers.Newznab            : [Search: 862915, IP: <IP>] Nzb.su: Calling https://www.nzb.su/api?apikey=<apikey>&t=movie&extended=1&cat=2000&limit=100&offset=0
2019-04-16 04:42:28.879  INFO --- [ool-67-thread-3] org.nzbhydra.indexers.Newznab            : [Search: 862915, IP: <IP>] dog: Calling https://api.dognzb.cr/api?apikey=<apikey>&t=search&extended=1&cat=2000&limit=100&offset=0
2019-04-16 04:42:28.879  INFO --- [ool-67-thread-4] org.nzbhydra.indexers.Newznab            : [Search: 862915, IP: <IP>] nzbgeek: Calling https://api.nzbgeek.info/api?apikey=<apikey>&t=search&extended=1&cat=2000&limit=100&offset=0
2019-04-16 04:42:28.879  INFO --- [ool-67-thread-2] org.nzbhydra.indexers.Newznab            : [Search: 862915, IP: <IP>] nzbplanet.net: Calling https://api.nzbplanet.net/api?apikey=<apikey>&t=search&extended=1&cat=2000&limit=100&offset=0
2019-04-16 04:42:29.174  INFO --- [ool-67-thread-2] org.nzbhydra.indexers.Newznab            : [Search: 862915, IP: <IP>] nzbplanet.net: Successfully executed search call in 281ms with 100 results
2019-04-16 04:42:30.187  INFO --- [ool-67-thread-3] org.nzbhydra.indexers.Newznab            : [Search: 862915, IP: <IP>] dog: Successfully executed search call in 1303ms with 100 results
2019-04-16 04:42:30.580  INFO --- [ool-67-thread-4] org.nzbhydra.indexers.Newznab            : [Search: 862915, IP: <IP>] nzbgeek: Successfully executed search call in 1693ms with 100 results
2019-04-16 04:42:30.905  INFO --- [ool-67-thread-1] org.nzbhydra.indexers.Newznab            : [Search: 862915, IP: <IP>] Nzb.su: Successfully executed search call in 2019ms with 100 results
2019-04-16 04:42:30.917  INFO --- [0.0-5076-exec-4] org.nzbhydra.searching.Searcher          : [Search: 862915, IP: <IP>] Returning results 1-100 from 380 results in cache. A total of 614421 results is available from indexers of which 0 were already rejected and 20 were removed as duplicates
2019-04-16 04:42:30.922  INFO --- [0.0-5076-exec-4] org.nzbhydra.api.ExternalApi             : [Search: 862915, IP: <IP>] Search took 2047ms. Returning 100 results
2019-04-16 04:43:39.523  INFO --- [0.0-5076-exec-1] org.nzbhydra.api.ExternalApi             : [IP: <IP>] Received external API call: NewznabParameters{t=MOVIE, cat=[2000], offset=100, limit=100, raw=false, o=XML, attrs=[], extended=true}
2019-04-16 04:43:39.523  INFO --- [0.0-5076-exec-1] org.nzbhydra.api.ExternalApi             : [Search: 843842, IP: <IP>] Executing new search
2019-04-16 04:43:39.524  INFO --- [0.0-5076-exec-1] org.nzbhydra.searching.Searcher          : [Search: 843842, IP: <IP>] Returning results 101-200 from 380 results in cache. A total of 614421 results is available from indexers of which 0 were already rejected and 0 were removed as duplicates
2019-04-16 04:43:39.534  INFO --- [0.0-5076-exec-1] org.nzbhydra.api.ExternalApi             : [Search: 843842, IP: <IP>] Search took 10ms. Returning 100 results
2019-04-16 04:44:58.189  INFO --- [0.0-5076-exec-4] org.nzbhydra.api.ExternalApi             : [Search: 862915, IP: <IP>] Received external API call: NewznabParameters{t=MOVIE, cat=[2000], offset=200, limit=100, raw=false, o=XML, attrs=[], extended=true}
2019-04-16 04:44:58.190  INFO --- [0.0-5076-exec-4] org.nzbhydra.api.ExternalApi             : [Search: 404258, IP: <IP>] Executing new search
2019-04-16 04:44:58.190  INFO --- [0.0-5076-exec-4] org.nzbhydra.searching.Searcher          : [Search: 404258, IP: <IP>] Returning results 201-300 from 380 results in cache. A total of 614421 results is available from indexers of which 0 were already rejected and 0 were removed as duplicates
2019-04-16 04:44:58.195  INFO --- [0.0-5076-exec-4] org.nzbhydra.api.ExternalApi             : [Search: 404258, IP: <IP>] Search took 5ms. Returning 100 results
2019-04-16 06:48:39.611  INFO --- [0.0-5076-exec-4] org.nzbhydra.searching.SearchWeb         : [Search: 966301, IP: <IP>] New search request: SearchRequest{source=INTERNAL, indexers=[nzbplanet.net], searchType=MOVIE, category=Movies, offset=0, limit=100, minsize=500, maxsize=20000, identifiers={}}
2019-04-16 06:48:39.614  INFO --- [0.0-5076-exec-4] o.n.searching.IndexerForSearchSelector   : [Search: 966301, IP: <IP>] Not using Nzb.su because it was not selected by the user
2019-04-16 06:48:39.614  INFO --- [0.0-5076-exec-4] o.n.searching.IndexerForSearchSelector   : [Search: 966301, IP: <IP>] Not using nzbgeek because it was not selected by the user
2019-04-16 06:48:39.614  INFO --- [0.0-5076-exec-4] o.n.searching.IndexerForSearchSelector   : [Search: 966301, IP: <IP>] Not using dog because it was not selected by the user
2019-04-16 06:48:39.614  INFO --- [0.0-5076-exec-4] o.n.searching.IndexerForSearchSelector   : [Search: 966301, IP: <IP>] Selected 1 out of 4 indexers: nzbplanet.net
2019-04-16 06:48:39.615  INFO --- [ool-73-thread-1] org.nzbhydra.indexers.Newznab            : [Search: 966301, IP: <IP>] nzbplanet.net: Calling https://api.nzbplanet.net/api?apikey=<apikey>&t=search&extended=1&minsize=500&cat=2000&limit=100&offset=0
2019-04-16 06:48:39.930  INFO --- [ool-73-thread-1] org.nzbhydra.indexers.Newznab            : [Search: 966301, IP: <IP>] nzbplanet.net: Successfully executed search call in 309ms with 100 results
2019-04-16 06:48:39.932  INFO --- [ool-73-thread-1] o.n.searching.SearchResultAcceptor       : [Search: 966301, IP: <IP>] Rejected 8 search results from nzbplanet.net for the following reason: Wrong size
2019-04-16 06:48:39.946  INFO --- [ool-76-thread-1] org.nzbhydra.indexers.Newznab            : [Search: 966301, IP: <IP>] nzbplanet.net: Calling https://api.nzbplanet.net/api?apikey=<apikey>&t=search&extended=1&minsize=500&cat=2000&limit=100&offset=100
2019-04-16 06:48:40.651  INFO --- [ool-76-thread-1] org.nzbhydra.indexers.Newznab            : [Search: 966301, IP: <IP>] nzbplanet.net: Successfully executed search call in 699ms with 100 results
2019-04-16 06:48:40.653  INFO --- [ool-76-thread-1] o.n.searching.SearchResultAcceptor       : [Search: 966301, IP: <IP>] Rejected 2 search results from nzbplanet.net for the following reason: Wrong size
2019-04-16 06:48:40.663  INFO --- [0.0-5076-exec-4] org.nzbhydra.searching.Searcher          : [Search: 966301, IP: <IP>] Returning results 1-189 from 189 results in cache. A total of 189114 results is available from indexers of which 10 were already rejected
2019-04-16 06:48:40.670  INFO --- [0.0-5076-exec-4] org.nzbhydra.searching.SearchWeb         : [Search: 966301, IP: <IP>] Search took 1058ms
2019-04-16 06:49:04.680  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Creating debug infos
2019-04-16 06:49:04.680  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] NZBHydra2 version: 2.5.2
2019-04-16 06:49:04.681  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Java command line: <BASEPATH>\lib\core-2.5.2-exec.jar --datafolder <BASEPATH>\data restarted
2019-04-16 06:49:04.681  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Java runtime name: Java(TM) SE Runtime Environment
2019-04-16 06:49:04.681  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Java runtime version: 1.8.0_201-b09
2019-04-16 06:49:04.681  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] OS name: Windows 10
2019-04-16 06:49:04.681  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] OS architecture: x86
2019-04-16 06:49:04.681  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] User country: GB
2019-04-16 06:49:04.681  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] File encoding: Cp1252
2019-04-16 06:49:04.682  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Number of rows in table SEARCH: 4282
2019-04-16 06:49:04.683  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Number of rows in table SEARCHRESULT: 8811
2019-04-16 06:49:04.683  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Number of rows in table INDEXERSEARCH: 17349
2019-04-16 06:49:04.684  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Number of rows in table INDEXERAPIACCESS: 41488
2019-04-16 06:49:04.684  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Number of rows in table INDEXERAPIACCESS_SHORT: 19325
2019-04-16 06:49:04.684  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Number of rows in table INDEXERNZBDOWNLOAD: 642
2019-04-16 06:49:04.687  INFO --- [0.0-5076-exec-4] o.n.debuginfos.DebugInfosProvider        : [Search: 966301, IP: <IP>] Size of database folder: 8MB

**Comment by jbravo2uk** [comment]
[nzbhydra-debuginfos-2019-04-16-06-46.zip](https://github.com/theotherp/nzbhydra2/files/3083299/nzbhydra-debuginfos-2019-04-16-06-46.zip)

**Comment by theotherp** [comment]
I don't have an account there so I can't test it. Are covers shown for nzbplanet? Are TV covers shown?

I didn't make any changes related to the cover display apart from the toggle in the display options being effective immediately. It might help if you just switch it off and on again...

**Comment by jbravo2uk** [comment]
If I toggle it makes no difference for movies or  TV this was working before I dont know what's changed if you PM me I will send you an invite to nzbplanet so you could test

**Comment by jbravo2uk** [comment]
Also on nzbplanet covers are shown for both on the site

**Comment by theotherp** [comment]
Please send an invite to theotherp@gmx.de

**Comment by jbravo2uk** [comment]
Sent

**Comment by theotherp** [comment]
Received :-)

**Comment by theotherp** [comment]
I was told in #357 that when searching NZBplanet using movie search without any query would return random results. Either that was fixed or it was never true. In any case, I changed the code to not use empty movie searches with nzbplanet but the indexer only returns covers in the results when movie searches are done.

I'll revert the change which should fix this issue. If you want to test this, make a movie search using an ID (i.e. select the checkbox, enter any title, select it from the dropdown). Covers should be shown.
